build.gradle

```diff
@@ -0,0 +1,29 @@
+allprojects {
+    apply plugin: "java"
+
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+        implementation group: 'org.springframework', name: 'spring-tx', version: springVersion
+        testImplementation group: 'org.springframework', name: 'spring-test', version: springVersion
+
+        implementation group: 'org.apache.commons', name: 'commons-dbcp2', version: dbcpVersion
+        implementation group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version: jacksonVersion
+        compileOnly("javax.servlet:javax.servlet-api:4.0.1")
+
+
+    }
+
+    tasks.withType(JavaCompile).each {

```

Comment:
What the purpose of this arg?

Reply:
I had problems with `"""..."""` quotation marks , needed it for building, I'll change the quotation marks to regular.